docs/tutorials/tictactoe.rst

```diff
@@ -162,8 +162,8 @@ Tianshou already provides some builtin classes for multi-agent learning. You can
 
 Random agents perform badly. In the above game, although agent 2 wins finally, it is clear that a smart agent 1 would place an ``x`` at row 4 col 4 to win directly. 
 
-Train a MARL Agent
-------------------
+Train an MARL Agent
```

Comment:
Why "an"?

Reply:
@Trinkle23897 reads it as M-A-R-L, so he uses "an". I read it as MA-R-L, so I use "a".